### DISCUSSION TITLE
建议给新用户增加运行环境的套餐选择和单一应用部署的参数提示

### DISCUSSION THREAD

**Discussion by yggcwx** [1Panel-dev/1Panel#4145] [Ideas]
1Panel面板现在并不提供运行环境的组合安装选项，需要用户在应用商店里一个一个装。
用面板的，要么是菜鸟，要么是高手不想麻烦了。
所以，提个兼顾熟手与生手的方案，供参考。

用户第一次登录面板，给出这样几个安装选项的提示：LNMP组合，LAMP组合，LNMPA组合+以上都不选、自行安装。
这样，菜鸟选套餐，高手自己弄，大家各取所需。

另外，部署常见应用时，选项往往很多、很令人纠结，希望系统能给出建议或提示，以及合理的默认选项。
这样，使用1Panel面板时小白和菜鸟就不会眼冒金星，无所适从了。

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，我们会在后续版本考虑上述建议并持续优化。
